iothub/device/tests/Mqtt/MqttIotHubAdapterTests.cs

```diff
@@ -0,0 +1,99 @@
+namespace Microsoft.Azure.Devices.Client.Test.Mqtt
```

Comment:
We already have a MqttIotHubAdapterTest.cs file and the class name must match the .cs file name. 
Could you please move this in the existing file instead of creating a new one?